src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowBuilder.java

```diff
@@ -447,5 +452,26 @@ String getReplacementVendorTaxNumber(String payeeId, T summary) {
         
         return stats;
     }
+    
+    protected int getMaxSearchSize() {
+    	if (maxSearchResultSize == null) {
+	    	String searchLimit = getParameterService().getParameterValueAsString(KewApiConstants.KEW_NAMESPACE, 
+	        		KRADConstants.DetailTypes.DOCUMENT_SEARCH_DETAIL_TYPE, KewApiConstants.DOC_SEARCH_RESULT_CAP);
+	    	try {
+	    		LOG.debug("Found a value for KewApiConstants.DOC_SEARCH_RESULT_CAP, and it is '" + searchLimit + "'");
+	    		maxSearchResultSize = Integer.parseInt(searchLimit);

```

Comment:
Implicit conversion of int to Integer, since Integer.parseInt() returns an int. Integer.valueOf() can be used to parse the string and return an Integer object instead. This code should still work fine as-is, but I would suggest updating this method to be consistent about implicit and explicit conversions, since an explicit one is occurring on the return statement.